oap-server/server-library/library-elasticsearch-client/src/main/java/org/apache/skywalking/library/elasticsearch/requests/search/aggregation/TermsAggregationBuilder.java

```diff
@@ -84,4 +99,34 @@ private ImmutableMap.Builder<String, Aggregation> subAggregations() {
         }
         return subAggregations;
     }
+
+    public enum CollectMode {
+        BREADTH_FIRST("breadth_first"), DEPTH_FIRST("depth_first");
+
+        final String value;
+
+        CollectMode(final String value) {
+            this.value = value;
+        }
+
+        @Override
+        public String toString() {
+            return value;
+        }
+    }
+
+    public enum ExecutionHint {
+        GLOBAL_ORDINALS("global_ordinals"), map("map");
```

Comment:
```suggestion
        GLOBAL_ORDINALS("global_ordinals"), MAP("map");
```

I think code style will fail due to lower case.